src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj

```diff
@@ -147,6 +147,10 @@
       <Name>PythonNodeModels</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\NodeServices\DynamoServices.csproj">
```

Comment:
whats added from this project?

Reply:
Analytics is only available from logging, so in the future, everytime we want to track on extension layer, this will be a dependency in that case seems